modules/kafka/examples_test.go

```diff
@@ -41,3 +49,337 @@ func ExampleRun() {
 	// test-cluster
 	// true
 }
+
+func ExampleKafkaContainer_BrokersByHostDockerInternal() {
+	ctx := context.Background()
+
+	kafkaContainer, err := kafka.Run(ctx,
+		"confluentinc/confluent-local:7.5.0",
+		kafka.WithClusterID("test-cluster"),
+	)
+	if err != nil {
+		log.Printf("failed to start container: %s", err)
+		return
+	}
+
+	// Clean up the container after
+	defer func() {
+		if err := kafkaContainer.Terminate(ctx); err != nil {
+			log.Fatalf("failed to terminate container: %s", err)
+		}
+	}()
+
+	{
```

Comment:
suggestion: no need for wrapping braces which make this harder to read, more below.